source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -139,7 +139,8 @@ def _direct_path(
         response = instance.send_and_receive(req, request_id=request_id)
         if should_invert_journey:
             return self._reverse_journeys(response)
-
+        if response.journeys:
+            response.journeys[0].sections[0].street_network.mode = FallbackModes[mode].value
```

Comment:
I don't understand this line ?

Reply:
this line gives a 'tag' to the returned direct_path. Notice that this line is located in 'kraken.py', a connector for distributed